docs/source/presentations.rst

```diff
@@ -1,5 +1,19 @@
 Presentations On Dask
 =====================
+* AMS 2018, January 2018
+
+  * `Dask and Xarray on the Cloud with JupyterHub
+    <https://www.youtube.com/watch?v=rSOJKbfNBNk>`__
```

Comment:
This is just a three minute screencast, of which there are several.  Perhaps the ESIP video would better serve here?

Reply:
@mrocklin, IMHO that video is super useful in that it is a complete demo in only 3 minutes, and it may get folks hooked to watch the more in-depth videos.   What do you think about adding the times, so that folks have some idea of what they are in for?   For example:

* Dask: Parallelism in Python (1 hour, 2 minutes)
* Dask and Xarray on the Cloud with JupterHub (3 minutes)